README.md

```diff
@@ -27,6 +28,38 @@ Rich IntelliSense (completions) for `Dockerfile` and `docker-compose.yml` files:
 
 ![IntelliSense for DockerFiles](images/intelliSense.gif)
 
+## Debugging (Preview)
+
+To debug a .NET Core web application running in a Linux Docker container, add a Docker .NET Core launch configuration:
+
+1. Switch to the debugging tab.
+1. Select `Add configuration...`
+1. Select `Docker: Launch .NET Core (Preview)`
+1. Set a breakpoint.
```

Comment:
It would be good to mention the prerequisites that will be required.

Reply:
Added note about dependency on C# VS Code extension.

Reply:
Doesn't it also require the az CLI?

Reply:
Yes, true.  Added a prerequisite check for the `dotnet` CLI.